styles.css

```diff
@@ -1,4 +1,65 @@
 /*Maak hier jouw CSS!*/
 #container {
+    background-color: #3B0D11;
+    padding: 8px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 50px 190px 100px auto 50px;
+    grid-template-areas:
+            "a c c"
+            "b c c"
+            "b e d"
+            "b e none"
```

Comment:
"none" mag ook ook opgelost worden met "." of ".................." het aantal puntjes maakt niet uit